# IDL struct ::niut::TIME_STR
uint32 t_sec
uint32 t_usec
